src/sentry/release_health/metrics_sessions_v2.py

```diff
@@ -425,13 +435,79 @@ def run_sessions_query(
     project_ids = filter_keys.pop("project_id")
     assert not filter_keys
 
-    orderby = _parse_orderby(query, fields)
-    if orderby is None:
-        # We only return the top-N groups, based on the first field that is being
-        # queried, assuming that those are the most relevant to the user.
-        # In a future iteration we might expose an `orderBy` query parameter.
-        primary_metric_field = _get_primary_field(list(fields.values()), query.raw_groupby)
-        orderby = OrderBy(primary_metric_field, Direction.DESC)
+    ordered_preflight_filters: Dict[GroupByFieldName, Sequence[str]] = {}
+    try:
+        orderby = _parse_orderby(query, fields)
```

Comment:
i know it was before, but i was wondering why don't you want to rename it to `order_by`?

Reply:
Do you mean `raw_orderby`?
tl;dr mypy complains 
Essentially `raw_orderby` is of type `str` while `orderby` is of type `OrderBy` (which is a dataclass)

Reply:
so from what i understood, you can't name it `order_by` because of mypy right?

Reply:
Yes

Reply:
gotcha.